Analysis/ElectronDresser/src/ElectronDresser.cc

```diff
@@ -0,0 +1,227 @@
+#include "ElectronDresser.h"
+
+const int MAX_COLL_NAME_WIDTH = 20;
+
+double DeltaR(double eta0, double phi0, double eta1, double phi1) {
+  Double_t deta = eta0 - eta1;
+  Double_t dphi = TVector2::Phi_mpi_pi(phi0 - phi1);
+  return TMath::Sqrt(deta * deta + dphi * dphi);
+}
+double EtaFromTheta(double theta) { return -TMath::Log(TMath::Tan(theta / 2)); }
+
+ElectronDresser aElectronDresser;
+
+ElectronDresser::ElectronDresser() : Processor("ElectronDresser") {
+  // modify processor description
+  _description = "ElectronDresser does whatever it does ...";
+
+  // register parameters: collection type, aribtrary name, arbitrary description, class-variable, default value
+  // input
+  registerInputCollection(LCIO::RECONSTRUCTEDPARTICLE, "", "", _m_pfos, std::string("LooseSelectedPandoraPFANewPFOs"));
```

Comment:
Can you add a description to your parameters please ? This is used by Marlin -x to document processor parameters.

Reply:
Done!